staging/src/k8s.io/apimachinery/pkg/labels/labels.go

```diff
@@ -54,9 +54,10 @@ func (ls Set) Has(label string) bool {
 	return exists
 }
 
-// Get returns the value in the map for the provided label.
-func (ls Set) Get(label string) string {
-	return ls[label]
+// Get returns the value and whether the provided label exists in the map.
+func (ls Set) Get(label string) (string, bool) {
+	ele, exists := ls[label]
+	return ele, exists
```

Comment:
you may return ls[label] directly like before.

Reply:
Hi, if use `ls[lable]` it will return value without whether the value exists, I'm not sure there is some way to do it.

Reply:
Hi, can you recheck it?

Reply:
ok